lib/Constants.vala

```diff
@@ -182,4 +182,80 @@ namespace Granite {
      * Transition duration when a widget opens, reveals more content, or enters the screen
      */
     public const int TRANSITION_DURATION_OPEN = 250;
+
+    /**
+    * Link to open Applications → Defaults settings page
+    */
+    public const string LINK_SETTINGS_APPLICATIONS_DEFAULTS = "settings://applications/defaults";
```

Comment:
Would it be better to put these constants under a dedicated namespace? Something like `Granite.Links`? (`Granite.Settings` is already "taken" by the settings class.)

Reply:
Yeah I think that would be better. My suggestion would be `Granite.SettingsUri` so that using it in practice would be like `new Gtk.UriLauncher (Granite.SettingsUri.LOCATION)` which seems consistent and succinct